app/build.gradle

```
@@ -14,7 +13,7 @@ android {
 		applicationId 'org.openobservatory.ooniprobe'
 		minSdkVersion 19
 		targetSdkVersion 29
-		versionName '2.7.0'
+		versionName '2.7.1'
```

```suggestion
		versionName '2.7.0'
```

It seems we don't need to release.